lib/src/screens/debug/scheme_management/scheme_manager_detail_screen.dart

```diff
@@ -0,0 +1,253 @@
+            : FlutterI18n.translate(context, 'debug.scheme_management.verify_pin.success'),
+      );
+    }
+  }
+
+  Future<void> _onActivateScheme(BuildContext context) async {
+    final language = FlutterI18n.currentLocale(context)?.languageCode ?? 'en';
+
+    final pin = await _requestPin(
+      context,
+      FlutterI18n.translate(context, 'debug.scheme_management.request_pin.title'),
+      FlutterI18n.translate(
+        context,
+        'debug.scheme_management.request_pin.content',
+        translationParams: {
+          'scheme': schemeManager.id,
+        },
+      ),
+    );
+    if (pin == null) return;
+
+    final navigator = Navigator.of(context);
+    final email = await navigator.push(
+      MaterialPageRoute(
+        builder: (context) => ProvideEmailScreen(
+          onEmailProvided: (email) => navigator.pop(email),
+          onEmailSkipped: () => navigator.pop(''),
+          onPrevious: () => navigator.pop(),
+        ),
+      ),
+    );
+    if (email == null) return;
+
+    navigator.pop();
+    showSnackbar(
+      context,
+      FlutterI18n.translate(
+        context,
+        'debug.scheme_management.activating',
+        translationParams: {
+          'scheme': schemeManager.id,
+        },
+      ),
+    );
+
+    final repo = IrmaRepositoryProvider.of(context);
+    repo.bridgedDispatch(EnrollEvent(
+      email: email,
+      pin: pin,
+      language: language,
+      schemeId: schemeManager.id,
+    ));
+
+    final event = await repo.getEvents().whereType<EnrollmentEvent>().first;
+    if (event is EnrollmentFailureEvent) {
+      navigator.push(MaterialPageRoute(
+        builder: (context) => ErrorScreen(
+          details: event.error.toString(),
+          onTapClose: () => navigator.pop(),
+        ),
+      ));
+    } else {
+      showSnackbar(
+        context,
+        FlutterI18n.translate(
+          context,
+          'debug.scheme_management.activate_success',
+          translationParams: {
+            'scheme': schemeManager.id,
+          },
+        ),
+      );
+    }
+  }
+
+  void _onDeleteScheme(BuildContext context) {
+    IrmaRepositoryProvider.of(context).bridgedDispatch(RemoveSchemeEvent(schemeId: schemeManager.id));
+    Navigator.of(context).pop();
+
+    showSnackbar(
+      context,
+      FlutterI18n.translate(
+        context,
+        'debug.scheme_management.remove',
+        translationParams: {
+          'scheme': schemeManager.id,
+        },
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final repo = IrmaRepositoryProvider.of(context);
+    final theme = IrmaTheme.of(context);
+
+    final isDeletable =
+        isActive && schemeManager.keyshareServer.isNotEmpty && schemeManager.id != repo.defaultKeyshareScheme;
+
+    Widget? bottomBar;
+    if (!isActive) {
+      bottomBar = IrmaBottomBar(
+        primaryButtonLabel: 'ui.activate',
+        onPrimaryPressed: () => _onActivateScheme(context),
+      );
```

Comment:
The activate button is also visible now when `isActive` is null. This causes the activate button to be visible for demo schemes, which don't need activation.